README.md

```diff
@@ -7,12 +7,12 @@ The site is made with [jekyll](https://jekyllrb.com/)
 ## Dependancies
 
 ```
-sudo gem install bundler jekyll
```

Comment:
Is sudo not still required for `gem install`? I think it might be?

Reply:
It's working without sudo on CI. Maybe a local setup thing?

Reply:
Ah, could be. Perhaps put a comment after, # this may need to be run as root, or even by specifying another install directory using e.g. `-n user/local/bin`

Reply:
I'm going to go read some Ruby docs to get this right.

Reply:
jekyll recommend `gem install --user-install bundler jekyll`